ui/tests/e2e/login.test.ts

```diff
@@ -20,7 +20,13 @@ describe('Login', () => {
 
       await ppExpect(page).toFill('input#tidb_signin_password', 'any')
       await ppExpect(page).toClick('button#signin_btn')
-      await ppExpect(page).toMatch('TiDB authentication failed')
+
+      const failReason = await page.waitForSelector(
+        'form#tidb_signin div[data-e2e="password"] div:last-child'
```

Comment:
This `div:last-child` is too magical.. Can we await for pages to contain some string?

Reply:
It seems no way to add the extra flag to the error message self except to its parent container likes `data-e2e="password"`.

Reply:
Or we can simply remove the `div:last-child`.

Reply:
Just got what you mean. I print the failed reason content to help debug what the exact failed reason. It may not be "TiDB authentication failed".

Reply:
Oh, got it!